10_rest/runnerdave/cmd/puppy_server/main.go

```diff
@@ -0,0 +1,94 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+
+	puppy "github.com/anz-bank/go-course/10_rest/runnerdave/pkg/puppy"
+	store "github.com/anz-bank/go-course/10_rest/runnerdave/pkg/puppy/store"
+
+	"github.com/go-chi/chi"
+	"gopkg.in/alecthomas/kingpin.v2"
+)
+
+var (
+	args           = os.Args[1:]
+	data           = kingpin.Flag("data", "data file").Short('d').Default("puppydata/data.json").ExistingFile()
+	port           = kingpin.Flag("port", "PORT").Short('p').Default("8080").String()
```

Comment:
nit: int? (should have noticed this earlier, apologies)

Reply:
fixed